pkg/ebpf/c/common/filtering.h

```diff
@@ -95,7 +95,8 @@ statfunc u64 equality_filter_matches(u64 filter_out_scopes, void *filter_map, vo
 {
     // check compute_scopes() for initial info
     //
-    // e.g.: cmdline: -f 2:comm=who -f 3:comm=ping -f 4:comm!=who
+    // hypothetical cmdline where the prefix numbers specify the scopes, e.g.:
+    //   -s 2:comm=who -s 3:comm=ping -s 4:comm!=who
```

Comment:
This command line (and below) should be updated. We don't have multi-scopes in the command line anymore.

Reply:
Yeah, that's why I changed it to hypothetical cmdline, but will think on a better way of explain the defined scopes.